<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1241</width>
    <height>698</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="bed_up_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>200</width>
     <height>110</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 20pt &quot;휴먼엑스포&quot;;</string>
   </property>
   <property name="text">
    <string>침대 위</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="glasses_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>200</width>
     <height>110</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 20pt &quot;휴먼엑스포&quot;;</string>
   </property>
   <property name="text">
    <string>안경</string>
   </property>
  </widget>
  <widget class="QPushButton" name="small_btn">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>250</y>
     <width>200</width>
     <height>110</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 20pt &quot;휴먼엑스포&quot;;</string>
   </property>
   <property name="text">
    <string>소변</string>
   </property>
  </widget>
  <widget class="QPushButton" name="big_btn">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>250</y>
     <width>200</width>
     <height>110</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 20pt &quot;휴먼엑스포&quot;;</string>
   </property>
   <property name="text">
    <string>대변</string>
   </property>
  </widget>
  <widget class="QPushButton" name="light_off_btn">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>200</width>
     <height>110</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 20pt &quot;휴먼엑스포&quot;;</string>
   </property>
   <property name="text">
    <string>소등</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bed_down_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>200</width>
     <height>110</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 20pt &quot;휴먼엑스포&quot;;</string>
   </property>
   <property name="text">
    <string>침대 아래</string>
   </property>
  </widget>
  <widget class="QPushButton" name="light_on_btn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>200</width>
     <height>110</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 20pt &quot;휴먼엑스포&quot;;</string>
   </property>
   <property name="text">
    <string>점등</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mistake_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>200</width>
     <height>110</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 20pt &quot;휴먼엑스포&quot;;</string>
   </property>
   <property name="text">
    <string>잘못보냈어요</string>
   </property>
  </widget>
  <widget class="QPushButton" name="window_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>200</width>
     <height>110</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 20pt &quot;휴먼엑스포&quot;;</string>
   </property>
   <property name="text">
    <string>창문</string>
   </property>
  </widget>
  <widget class="QPushButton" name="out_btn">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>370</y>
     <width>200</width>
     <height>110</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 20pt &quot;휴먼엑스포&quot;;</string>
   </property>
   <property name="text">
    <string>나가고싶어요</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cold_btn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>490</y>
     <width>200</width>
     <height>110</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 20pt &quot;휴먼엑스포&quot;;</string>
   </property>
   <property name="text">
    <string>추워요</string>
   </property>
  </widget>
  <widget class="QPushButton" name="water_btn">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>370</y>
     <width>200</width>
     <height>110</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 20pt &quot;휴먼엑스포&quot;;</string>
   </property>
   <property name="text">
    <string>물</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hot_btn">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>490</y>
     <width>200</width>
     <height>110</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 20pt &quot;휴먼엑스포&quot;;</string>
   </property>
   <property name="text">
    <string>더워요</string>
   </property>
  </widget>
  <widget class="QPushButton" name="full_btn">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>490</y>
     <width>200</width>
     <height>110</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 20pt &quot;휴먼엑스포&quot;;</string>
   </property>
   <property name="text">
    <string>배불러</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hungry_btn">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>490</y>
     <width>200</width>
     <height>110</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 20pt &quot;휴먼엑스포&quot;;</string>
   </property>
   <property name="text">
    <string>배고파</string>
   </property>
  </widget>
  <widget class="QPushButton" name="emergency_btn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>370</y>
     <width>411</width>
     <height>110</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>휴먼엑스포</family>
     <pointsize>25</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
font: 25pt &quot;휴먼엑스포&quot;;</string>
   </property>
   <property name="text">
    <string>비상호출</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>411</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;메세지 출력 :&lt;/span&gt; &lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cam_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>637</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(217, 223, 255);
font: 9pt &quot;휴먼엑스포&quot;;</string>
   </property>
   <property name="text">
    <string>CAM </string>
   </property>
  </widget>
  <widget class="QPushButton" name="my_btn">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>90</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="image/icon.qrc">
     <normaloff>:/my/my.png</normaloff>:/my/my.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>90</width>
     <height>100</height>
    </size>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="setting_btn">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>270</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="image/icon.qrc">
     <normaloff>:/setting/setting.png</normaloff>:/setting/setting.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>90</width>
     <height>100</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="gallery_btn">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>450</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="image/icon.qrc">
     <normaloff>:/gallery/gallery.png</normaloff>:/gallery/gallery.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>90</width>
     <height>100</height>
    </size>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_btn">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>130</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(217, 223, 255);
font: 9pt &quot;휴먼엑스포&quot;;</string>
   </property>
   <property name="text">
    <string>삭제</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="image/icon.qrc"/>
 </resources>
 <connections/>
</ui>
